AWS/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = []
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.12"
-sekoia-automation-sdk = { version="1.5.2", extras=["all"] }
+sekoia-automation-sdk = "^1.8.1"
```

Comment:
Won't it create an issue with async libs not being installed?

Reply:
Yes. Fixed.

Reply:
Arf. I forgot the lock :/